void/PKGBUILD

```
@@ -9,7 +9,8 @@ pkgdesc="The open-source Cursor alternative"
 url="https://voideditor.com/"
 arch=('x86_64')
 license=("MIT")
-options=(!debug) # probably will be quite big
+options=(!strip # for sign of extension ?
+!debug) # probably big
```

???????? what the fuck?
```suggestion
options=('!strip' '!debug')
```

---

How about comment? Some one may tries to remove them again.

---

Comment on top properly then, with an explanation, not in-between and with cryptic one-liners. These comments mean nothing by themselves to any third party.

---

who? or why? "probably big" isn't a good comment. I doubt anyone will ever touch pacman options here, and the commit history will speak for itself on why this was added in the first place, so your concern is unfounded.